### PR TITLE
V4 output update

### PR DESCRIPTION
**Pull request by SEAJang**
# Pull Request Key Information

## Reason for change

Summary output plots were difficult to understand when they were based on using the total emissions of the full duration of the simulation

## What was changed

Summary plots axis titles were updated and changed to report annualized data instead

## Intended Purpose

Improve the usability of the summary plots

## Testing Completed

Manual walkthrough of the values and outputs and added relevant unit tests.

### CODE REVIEW

File: LDAR_Sim/src/file_processing/output_processing/multi_simulation_visualizations.py [hunk 1]
```diff
@@ -27,6 +28,67 @@
 from matplotlib import ticker, lines
 
 from file_processing.output_processing import output_constants, output_utils
+import numpy as np
+
+Y_AXIS_LABEL = "Likelihood of Occurrence (%)"
```
Comment (ThomasGalesloot) [inline LDAR_Sim/src/file_processing/output_processing/multi_simulation_visualizations.py:33]: Can these get moved to wherever we are keeping the rest of the constants for outputs?

File: LDAR_Sim/src/file_processing/output_processing/multi_simulation_visualizations.py [hunk 2]
```diff
@@ -27,6 +28,67 @@
...
+    return [
+        program_name
+        for program_name in emis_summary_info[
+            output_constants.EMIS_SUMMARY_COLUMNS_ACCESSORS.PROG_NAME
+        ].unique()
+        if program_name != baseline_program
+    ]
+
+
+def gen_annual_emissions_summary_list(emis_summary_info, program_names) -> dict[str, list]:
```
Comment (ThomasGalesloot) [inline LDAR_Sim/src/file_processing/output_processing/multi_simulation_visualizations.py:47]: Note: There could be value in testing this method specifically

File: LDAR_Sim/src/file_processing/output_processing/multi_simulation_visualizations.py [hunk 3]
```diff
@@ -126,31 +190,18 @@ def gen_estimated_vs_true_emissions_percent_difference_plot(
     combine_plots: bool = False,
     save_separate_plots: bool = True,
 ):
+    X_AXIS_LABEL = 'Percent Difference between "Estimated" and "True" Emissions'
```
Comment (ThomasGalesloot) [inline LDAR_Sim/src/file_processing/output_processing/multi_simulation_visualizations.py:193]: Nit: Move this with the other output constants

File: LDAR_Sim/src/file_processing/output_processing/multi_simulation_visualizations.py [hunk 4]
```diff
@@ -220,31 +271,20 @@ def gen_estimated_vs_true_emissions_relative_difference_plot(
...
-    )
-
-    program_names = [
-        program_name
-        for program_name in emis_summary_info[
-            output_constants.EMIS_SUMMARY_COLUMNS_ACCESSORS.PROG_NAME
-        ].unique()
-        if program_name != baseline_program
-    ]
+    X_AXIS_LABEL = "Relative Difference between Estimated and True Emissions"
```
Comment (ThomasGalesloot) [inline LDAR_Sim/src/file_processing/output_processing/multi_simulation_visualizations.py:274]: Nit: Move this with the other constants

File: LDAR_Sim/src/file_processing/output_processing/multi_simulation_visualizations.py [hunk 5]
```diff
@@ -321,28 +361,20 @@ def gen_paired_estimate_and_true_emission_distributions(
     combine_plots: bool = False,
     save_separate_plots: bool = True,
 ):
+    X_AXIS_LABEL = "Annual Emissions of All Sites (Kg Methane)"
```
Comment (ThomasGalesloot) [inline LDAR_Sim/src/file_processing/output_processing/multi_simulation_visualizations.py:364]: Nit: Move this with the other constants

**Review by ThomasGalesloot — APPROVED** [review]
Great work - Just a few minor nitpicks to address before merging. Also with these types of PR's in the future, I think it would be good to include images of the new outputs for review in the PR.

**Comment by SEAJang** [comment]
Example Figures. 
P_OGI True_vs_Estimated_Emissions_relative_differences
P_OGI True_and_Estimated_Paired_Emissions_Distribution
P_OGI True_vs_Estimated_Emissions_percent_differences

File: LDAR_Sim/src/file_processing/output_processing/multi_simulation_outputs.py [hunk 6]
```diff
@@ -39,6 +39,22 @@ def get_sum(df: pd.DataFrame, column: str) -> float:
     return df[column].sum()
 
 
+def assign_part_sum_columns(df: pd.DataFrame, column_name: str, base_column_name: str):
+    # Extracting distinct years from the 'Date Began' column
+    distinct_years = pd.to_datetime(
+        df[output_utils.EMIS_DATA_COL_ACCESSORS.DATE_BEG]
+    ).dt.year.unique()
+    result = {}
+    for year in distinct_years:
```
[the remark this answers is not in the document]
Comment (SEAJang) [reply LDAR_Sim/src/file_processing/output_processing/multi_simulation_outputs.py:48]: I agree. And this will have to come with the Document changes.

**Comment by SEAJang** [comment]
P_OGI True_and_Estimated_Paired_Emissions_Distribution
P_OGI True_vs_Estimated_Emissions_percent_differences
P_OGI True_vs_Estimated_Emissions_relative_differences

**Pull request merged**